### ISSUE TITLE
resolve-url-loader tries to find http urls

### ISSUE THREAD

**Issue by rdhaliwal** [bholloway/resolve-url-loader#66]
Hi there! Love your work 😄 

One issue I encountered was that resolve-url-loader was trying to find 'http' urls in css. In our css, we have something like:

```css
.HomeCTA-container {
  background-size: cover;
  background-image: url("https://res.cloudinary.com/yourgrocer/image/upload/c_scale,q_52,w_1920,f_auto/v1499829754/home/homepage-hero1_d7lj8c.jpg");
}
```

And I noticed that webpack was taking forever to compile. After debugging it a bit, it was scouring every folder trying to find this. While it ignored node_modules, we have a ruby app, so it was going through all the ruby gems we have installed. My workaround was to specify the 'root' as `app/assets`, so it gave up if it couldn't find it there. 

I guess I'm raising this issue because I believe it should ignore http urls, and only try and follow relative paths/files.

I'm using:
```
"webpack": "^3.3.0",
"resolve-url-loader": "^2.1.0",
yarn 0.27.5
node 8.1.4
```

**Comment by bholloway** [comment]
@rdhaliwal sorry for the delay on this. Please refer to #68.

**Comment by bholloway** [comment]
Published as `resolve-url-loader@2.1.1`.

Please let me know if you have any problems @rdhaliwal and I will reopen.

**Comment by rdhaliwal** [comment]
No worries, thanks for the fix!
